conanfile.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os.path
+from conans import ConanFile, CMake
+from conans.tools import Git
+from conans.errors import ConanInvalidConfiguration
+
+class ImguiSFML(ConanFile):
+    """ Imgui-SFML library conan file.
+
+    Options:
+
+    * shared: True or False
+      Whether to build this library shared or static
+
+    * fPIC: True or False
```

Comment:
Is this still needed?

Reply:
`fPIC` option ? I've seen it in every library recipe I used so far, and just like `shared` option, it is automatically managed with CMake.

Reply:
I see. Seems like Conan passes it automatically somehow..,